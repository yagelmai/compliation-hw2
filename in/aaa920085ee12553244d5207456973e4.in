
void main()
{


    if (a) if (b) s; else s2;

}
