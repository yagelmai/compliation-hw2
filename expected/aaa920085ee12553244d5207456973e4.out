6: RetType ->  VOID
7: Formals -> epsilon
35: Exp -> ID
line 6: syntax error
